app/conf/PaperPlans.scala

```diff
@@ -4,29 +4,29 @@ import com.typesafe.config.Config
 import wiring.AppComponents.Stage
 
 case class PaperProducts(
-  digitalVoucher: PaperPlans,
-  voucher: PaperPlans,
-  delivery: PaperPlans,
-  nationalDelivery: NationalDeliveryPaperPlans
+    digitalVoucher: PaperPlans,
+    voucher: PaperPlans,
+    delivery: PaperPlans,
+    nationalDelivery: NationalDeliveryPaperPlans,
 )
 
 case class PaperPlans(
-  saturday: ProductRatePlanId,
-  saturdayplus: ProductRatePlanId,
-  sunday: ProductRatePlanId,
-  sundayplus: ProductRatePlanId,
-  weekend: ProductRatePlanId,
-  weekendplus: ProductRatePlanId,
-  sixday: ProductRatePlanId,
-  sixdayplus: ProductRatePlanId,
-  everyday: ProductRatePlanId,
-  everydayplus: ProductRatePlanId
```

Comment:
Is there a reason to keep plus and non-plus rate plans here? Could we lose the non-plus ones (except sunday)?